semanticTable.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="mystyle.css" />
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="code example for html, css and semantic tags">
+    <meta name="keywords" content="HTML, CSS, semantic-tag">
+    <meta name="author" content="Carlos Isaac Jaldin Benavides and Giovanni Alcocer">
+    <base href="https://www.w3schools.com" target="_blank">
+    <link rel="icon" href="https://upload.wikimedia.org/wikipedia/commons/4/47/PNG_transparency_demonstration_1.png">
+    <title>
+        Exercise 1
+    </title>
+</head>
+    <body>
+        <header>
+            <h1>HTML Semantics and Flexbox</h1>
+        </header>
+        <main>
```

Comment:
Not sure if the main tag should include the nav, main content and section but not a big deal for me.

Reply:
Already checked, I'm not pretty sure too but let's check tomorrow. I'm also saw that there are many people who set the navbar at the header but in this case header are not above... In conclusion I'm willing to asume the consequences if you are agree